test/node/byte_utils.test.ts

```diff
@@ -544,6 +566,32 @@ describe('ByteUtils', () => {
     });
   });
 
+  describe('randomBytes fallback case', () => {
+    let bsonWithNoCrypto;
+    before(function () {
+      bsonWithNoCrypto = loadBSONWithGlobal({
+        crypto: null,
+        // if we don't add a copy of Math here then we cannot spy on it for the test
+        Math: {
+          pow: Math.pow,
+          floor: Math.floor,
+          random: Math.random
+        }
+      });
+    });
```

Comment:
Isn't there different behavior here for both web and nodejs?  NodeJS falls back, web only falls back if it is not react native.  Should we add platform specific tests for these behaviors somewhere (if we don't have them)?

Reply:
Added tests for nodejs and react native